src/widgets/modalChildren/PatientEditModal.js

```diff
@@ -89,11 +89,18 @@ export const PatientEditModalComponent = ({
           />
         </View>
       </FlexRow>
+      <PaperModalContainer isVisible={cantDeleteModalOpen} onClose={toggleCantDeleteModal}>
```

Comment:
nit: maybe it's better to use something like `cannotDeleteModalOpen` and `toggleCannotDeleteModal` instead of `cantDeleteModalOpen` and `toggleCantDeleteModal`?

Reply:
Resolved with bc81d43f